vuepress/docs/v7.0/tutorials/create/mfe/widget-configuration.md

```diff
@@ -1,24 +1,24 @@
 
 # Add a Configuration Screen in App Builder
 
-Entando widgets can be customized through an App Builder configuration screen that is itself a micro frontend.
+Entando widgets can be customized through an App Builder configuration screen that is itself a micro frontend. This tutorial splits the process into 3 steps:
```

Comment:
tried "phases" and "successive stages" and considered bullets or letters instead of a numbered list, but then decided "steps" was the clearest way to describe performing a sequence of actions.